model/validation.go

```diff
@@ -669,3 +661,89 @@ func ValidateDestinationPolicy(msg proto.Message) error {
 
 	return errs
 }
+
+func validateProxyAddress(hostAddr string) error {
```

Comment:
Not sure what you suggest. "my-name" and "10.0.0.10" are both valid envoy hosts. This will just detect the second case.